xml/hwbp_registers.xml

```diff
@@ -242,7 +251,7 @@
             1: Perform a match on the data value loaded or stored, or the
             instruction executed.
         </field>
-        <field name="timing" bits="18" access="R/W" reset="0">
+        <field name="timing" bits="18" access="WARL" reset="0">
```

Comment:
If you're going to change this to WARL, why not the rest of the fields in this register?  They can only take on certain legal values too.

Reply:
Perhaps maskmax is the only field not WARL, because it's not writable.

Reply:
Good point. I changed almost all of them.